environment/src/main/java/com/sequenceiq/environment/credential/attributes/azure/AzureCredentialCertificate.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.environment.credential.attributes.azure;
+
+import com.sequenceiq.common.api.credential.AppCertificateStatus;
+
+public class AzureCredentialCertificate {
+
+    private AppCertificateStatus status;
+
+    private Long expiration;
+
+    private String certificate;
+
+    private String sha512;
```

Comment:
same here: based on the field name I am not sure what is this for.

Reply:
same as above